requirements-common.txt

```diff
@@ -19,7 +19,7 @@ prometheus_client >= 0.18.0
 prometheus-fastapi-instrumentator >= 7.0.0
 tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.9, < 0.11
-outlines == 0.1.9
+outlines == 0.1.10
```

Comment:
Are there aarch64 manylinux wheels now for outlines-core?

Reply:
i haven't checked yet ...

Reply:
I bumped this PR up to outlines-0.1.11 which should have arm builds now

https://github.com/dottxt-ai/outlines-core/issues/122